src/main/resources/spotify_checks.xml

```diff
@@ -164,11 +164,13 @@
         </module>
         <module name="OverloadMethodsDeclarationOrder"/>
         <module name="VariableDeclarationUsageDistance"/>
-        <module name="CustomImportOrder">
-            <property name="specialImportsRegExp" value="com.spotify"/>
-            <property name="sortImportsInGroupAlphabetically" value="true"/>
-            <property name="customImportOrderRules" value="STATIC###SPECIAL_IMPORTS###THIRD_PARTY_PACKAGE###STANDARD_JAVA_PACKAGE"/>
-        </module>
+        <module name="ImportOrder">
+            <property name="option" value="above"/>
```

Comment:
I think this should be `top`: http://checkstyle.sourceforge.net/property_types.html#importOrder

> Imports are ordered as follows:
>
> All static imports in a single block.
> All non-static imports in a single block.

https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing